.demo/env/charts/gatekeeper.yaml

```diff
@@ -0,0 +1,75 @@
+charts:
+    gatekeeper:
+        enabled: false
+        constraints:
+            - parameters:
+                enabled: false
+                cpu: 10m
+                memory: 120Mi
+              policyName: containerlimits
+            - parameters:
```

Comment:
this policy will not be used as it is unmanageable and the problem it tries to solve is better solved by another solution, like Harbor